index.js

```diff
@@ -11,16 +11,46 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+const maxPage = 42;
+let page = 1;
 const searchQuery = "";
 
 // Functions
 
 async function fetchCharacters() {
-  const response = await fetch("https://rickandmortyapi.com/api/character");
+  const response = await fetch(
+    `https://rickandmortyapi.com/api/character?page=${page}`
+  );
   const json = await response.json();
-  return json.results;
+  cardContainer.innerHTML = "";
+  const pageChange = `<span class="navigation__pagination" data-js="pagination">${page} / ${maxPage}</span>`;
+  pagination.innerHTML = pageChange;
+
+  return json.results.forEach((character) => {
+    createCharacterCard(character);
```

Comment:
If you changed the `createCharacterCard` function as I mentioned above, the function will return the created `li`-element. You need to save the return value inside a variable. Additionally, you need to append the created `li`-element to the `cardContainer`.

```suggestion
 characters.forEach((character) => {
    const card = createCharacterCard(character);
    cardContainer.append(card);
  });
```